streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceiverTracker.scala

```diff
@@ -137,8 +141,24 @@ class ReceiverTracker(ssc: StreamingContext, skipReceiverLaunch: Boolean = false
     }
     receiverInfo(streamId) = ReceiverInfo(
       streamId, s"${typ}-${streamId}", receiverActor, true, host)
+    logInfo(s"Receiver for stream ${streamId} from ${sender.path.address} " +
+      s"attempts to register.")
+  }
+
+  /** Report a starting receiver */
+  private def startingReceiver(
```

Comment:
Ah, I see. I will close the PR first and take a close look. Thanks for your review